src/org/labkey/test/pages/ReactAssayTypeSelectPage.java

```diff
@@ -0,0 +1,86 @@
+package org.labkey.test.pages;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ReactAssayTypeSelectPage<EC extends ReactAssayTypeSelectPage.ElementCache> extends WebDriverComponent<EC>
+{
+    private final WebElement el;
+    private final WebDriver driver;
+
+    public ReactAssayTypeSelectPage(WebDriver driver)
+    {
+        this.driver = driver;
+        el = Locator.id("app").findElement(this.driver); // Full page component
+    }
+
+    @Override
+    public WebElement getComponentElement()
+    {
+        return el;
+    }
+
+    @Override
+    public WebDriver getDriver()
+    {
+        return driver;
+    }
+
+    public ReactAssayDesignerPage selectStandardAssay()
+    {
+        getWrapper().click(elementCache().stdAssayTab);
+        getWrapper().clickAndWait(elementCache().saveButton);
```

Comment:
Be cautious when using clickAndWait in react components, if this will be used in an app environment we can't always count on a page load event happening

Reply:
I added a TODO for now.  We can check this in the next story, integrating this component into Biologics